packages/phone-number-privacy/signer/src/database/wrappers/domainRequest.ts

```diff
@@ -0,0 +1,68 @@
+import { DB_TIMEOUT, Domain, domainHash, ErrorMessage } from '@celo/phone-number-privacy-common'
+import Logger from 'bunyan'
+import { Transaction } from 'knex'
+import { Counters, Histograms, Labels } from '../../common/metrics'
+import { getDatabase } from '../database'
+import {
+  DomainRequestRecord,
+  DOMAIN_REQUESTS_COLUMNS,
+  DOMAIN_REQUESTS_TABLE,
+} from '../models/domainRequest'
+
+function domainRequests<D extends Domain>() {
+  return getDatabase()<DomainRequestRecord<D>>(DOMAIN_REQUESTS_TABLE)
+}
```

Comment:
Nit: I would argue this method is not actually generic because the return value of this function does not depend on the type parameter (nor can it as a rule in TypeScript). If we have multiple domain types in the future, this database will either hold the requests for all domain types since the columns are agnostic to the actual domain type underneath.
```suggestion
function domainRequests() {
  return getDatabase()<DomainRequestRecord<Domain>>(DOMAIN_REQUESTS_TABLE)
}
```